fdbserver/ResolverInterface.h

```diff
@@ -77,10 +77,11 @@ struct ResolveTransactionBatchReply {
 	VectorRef<uint8_t> committed;
 	Optional<UID> debugID;
 	VectorRef<VectorRef<StateTransactionRef>> stateMutations;  // [version][transaction#] -> (committed, [mutation#])
+	Optional<std::map<int, std::vector<int>>> conflictingKeyRangeMap; // transaction index -> conflicting read_conflict_range ids given by the resolver
 
 	template <class Archive>
 	void serialize(Archive& ar) {
-		serializer(ar, committed, stateMutations, arena, debugID);
+		serializer(ar, committed, stateMutations, arena, debugID, conflictingKeyRangeMap);
```

Comment:
Can you try to make `conflictingKeyRangeMap` a map of `VectorRef` again and then move the arena to the end (ie make it the last argument to `serializer`)? This might fix the problem you saw before.

Arenas and serializers are a bit weird, and with flatbuffers Arena is the only type you are allowed to move around without breaking wire-compatability.

Reply:
It solved the problem!
Changed back to VectorRef.